src/assets/main.scss

```diff
@@ -16,6 +16,45 @@ body {
   text-rendering: optimizeLegibility;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  margin-bottom: 50px;
```

Comment:
Consider grouping all `.scss` files in `styles` subfolder, similar to what you have with `images`